.github/workflows/buildMLTBXTags.yml

```diff
@@ -10,8 +11,8 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "MLTBXBuild"
-  MLTBXBuildTag:
-    # The type of runner that the job will run on
+  MLTBXBuild:
+    # The type of runner that the job will run on, we use ubuntu
```

Comment:
use the OS name as officially used "Ubuntu". Do we define it to be Ubuntu?

Reply:
Replaced ubuntu with Ubuntu.